src/main/resources/db/migration/V3__Create_klage_schema.sql

```diff
@@ -1,34 +1,61 @@
+CREATE TYPE id_type AS ENUM ('PERSON', 'ORGANISASJON', 'VIRKSOMHET');
+
+CREATE TABLE klage.part_id
+(
+    id          UUID    PRIMARY KEY,
+    type        id_type NOT NULL,
+    value       TEXT
+);
+
 CREATE TABLE klage.mottak
 (
-    id                               UUID PRIMARY KEY,
-    versjon                          BIGINT                   NOT NULL,
-    tema_id                          VARCHAR(3)               NOT NULL,
-    sakstype_id                      VARCHAR(10)              NOT NULL,
-    referanse_id                     TEXT,
-    innsyn_url                       TEXT,
-    foedselsnummer                   VARCHAR(11),
-    organisasjonsnummer              VARCHAR(9),
-    virksomhetsnummer                VARCHAR(9),
-    hjemmel_liste                    TEXT,
-    beskrivelse                      TEXT,
-    avsender_saksbehandlerident      VARCHAR(50),
-    avsender_enhet                   VARCHAR(10),
-    oversendt_klageinstans_enhet     VARCHAR(10),
-    oversendelsesbrev_journalpost_id VARCHAR(40),
-    brukers_klage_journalpost_id     VARCHAR(40),
-    dato_innsendt                    DATE,
-    dato_mottatt_foersteinstans      DATE,
-    dato_oversendt_klageinstans      DATE                     NOT NULL,
-    dato_frist_fra_foersteinstans    DATE,
-    kilde                            VARCHAR(50)              NOT NULL,
-    created                          TIMESTAMP WITH TIME ZONE NOT NULL,
-    modified                         TIMESTAMP WITH TIME ZONE NOT NULL,
+    id                                  UUID PRIMARY KEY,
+    versjon                             BIGINT                   NOT NULL,
+    tema_id                             VARCHAR(3)               NOT NULL,
+    sakstype_id                         VARCHAR(10)              NOT NULL,
+    klager_part_id                      UUID,
+    saksreferanse                       TEXT,
+    intern_referanse                    TEXT,
```

Comment:
Dette er den for å dyplenke?

Reply:
innsyn_url er full dyplenke. Dette er bare referansen vi kan konstruere dyplenke fra.